extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -819,6 +819,11 @@ export class ProjectsController {
 			return; // user cancelled
 		}
 
+		// Check if itemObjectName contains the file extension, remove it if so
+		while (itemObjectName.toLowerCase().endsWith(fileExtension.toLowerCase())) {
+			itemObjectName = itemObjectName?.slice(0, -fileExtension.length).trim();
```

Comment:
Why remove the extension only to add it again?  Seems like the clean fix is to just do:

```
const fileName = itemObjectName.endsWith(extension) ? itemObjectName : itemObjectName + extension;
const relativeFilePath = path.join(relativePath, fileName);
```

Reply:
we need the itemObjectName without the extension, for using it in the script " Create Table [TableName]", otherwise the file will create with the Table.sql only, but the script will be like "Create Table [Table.Sql]".